drizzlepac/tweakreg.help

```diff
@@ -647,17 +642,12 @@ Save the residual offsets (``dx``, ``dy``, ``rot``, ``scale``, ``xfit_rms``, ``y
 
     >>> tweakreg.edit_imagefindpars()
 
-3. Help can be accessed via the "Help" pulldown menu in the TEAL GUI.  It can also
-   be accessed from the PyRAF command-line and saved to a text file:
+2. Help can be accessed via the "Help" pulldown menu in the TEAL GUI.  It can also
+   be accessed from the python command-line and saved to a text file:
```

Comment:
The text has been removed which actually shows the help information being written to a file.